packages/itwinui-css/src/tile/tile.scss

```diff
@@ -189,10 +191,15 @@
     > svg,
     .iui-tile-metadata-icon {
       @include iui-icon-style('m');
-      @include iui-icon-color;
+      fill: var(--iui-color-icon-muted);
       margin-right: var(--iui-size-xs);
     }
 
+    .iui-tag-container,
+    .iui-tag-basic {
+      color: inherit;
```

Comment:
I think these should be moved under disabled styling

Reply:
Done.